src/multivariate/multinomial.jl

```diff
@@ -23,18 +23,21 @@ struct Multinomial{T<:Real} <: DiscreteMultivariateDistribution
     n::Int
     p::Vector{T}
 
-    function Multinomial{T}(n::Integer, p::Vector{T}) where T
+    Multinomial{T}(n::Integer, p::Vector{T}) where {T} = new{T}(round(Int, n), p)
```

Comment:
```suggestion
    Multinomial{T}(n::Integer, p::Vector{T}) where {T} = new{T}(Int(n), p)
```
But is that constructor even needed?